### ISSUE TITLE
Model is not working, csv training

### ISSUE THREAD

**Issue by SiBensberg** [xuannianz/EfficientDet#129]
Hi, :)
today I started a new Training with my own csv data. I pulled the latest version of the repository and installed all requierements.txt into a fresh, untouched anaconda environment. I have to use tf 1.15 because the cluster I am training on does not support cuda 10.2 yet and I could not train with tf 2.1 and a cuda <10.2 
The Problem is "inference.py" is not detecting Objects. There are Output boxes:
![grafik](https://user-images.githubusercontent.com/55738346/78027708-17b3c980-735e-11ea-8a57-8b6a27401e10.png)
But all Labels are "0" and the score is "0.01000005" for every box. They are not even near any plausible detection. 

I trained my Model with:
`srun python3 train.py --snapshot imagenet --phi 1 --gpu 0,1 --random-transform --compute-val-loss --freeze-backbone --epochs 200 --batch-size 24 --steps 1000 csv /work/traffic_dataset/train_labels_eff.csv /work/traffic_dataset/class_name_to_ID.csv --val-annotations-path /work/traffic_dataset/validation_labels_eff.csv`

with 

- cuda/10.0

- lib/cudnn/7.6.1.34_cuda_10.0

What makes me think something is wrong with my training is, the model started with an accuracy of 0.38 with an Imagenet snapshot. I don't think this is legit. I am trying to detect traffic obstacles.
![grafik](https://user-images.githubusercontent.com/55738346/78021639-df0ef280-7353-11ea-80bf-3a1b511c22bc.png)

I changed one line in the csv.py:
```
            if osp.exists(csv_data_file):
                self.base_dir = ''
            else:
                self.base_dir = osp.dirname(csv_data_file)
```
**to**
```
        if self.base_dir is None:
            self.base_dir = osp.dirname(csv_data_file)
```
because the images could not be found else.

Inference.py:
```
    phi = 1
    weighted_bifpn = False
    model_path = 'checkpoints/2020-03-30/csv_08_0.1805_0.2343.h5'
    image_sizes = (512, 640, 768, 896, 1024, 1280, 1408)
    image_size = image_sizes[phi]
    # coco classes
    classes = ['blue', 'yellow', 'orange', 'orange_big', ] 
```
It works good with the coco weights provided.

Do you have any guess what is the Problem?

Best regards
Simon

**Comment by SiBensberg** [comment]
i additonally noticed, the number of false positives while validation is very high and the number of true positives is about half the annotations (28046):
`num_fp=248701.0, num_tp=14564.0`

**Comment by SiBensberg** [comment]
this is is still a Problem. I am still using old code
